<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiPage</class>
 <widget class="QDialog" name="WifiPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 38, 43);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>热点设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
alternate-background-color:  &quot;#2a2c31&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/icon_back.png</normaloff>:/img/icon_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>33</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>72</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot; #232429&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addWifi">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>0</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
alternate-background-color:  &quot;#2a2c31&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/icon_add.png</normaloff>:/img/icon_add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>33</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="dialog" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>73</y>
     <width>540</width>
     <height>887</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2effffff&quot;;</string>
   </property>
   <widget class="QLineEdit" name="pswEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>425</y>
      <width>391</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>67</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:&quot;#00000000&quot;;
 </string>
    </property>
    <property name="text">
     <string>名称:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>67</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color:&quot;#00000000&quot;;</string>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>345</y>
      <width>391</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>507</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="commitButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>507</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>501</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 38, 43);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>nameEdit</zorder>
   <zorder>pswEdit</zorder>
   <zorder>cancelButton</zorder>
   <zorder>commitButton</zorder>
  </widget>
  <widget class="MyListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>73</y>
     <width>540</width>
     <height>887</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(37, 38, 43);
selection-color: rgb(37, 38, 43);
selection-background-color: rgb(37, 38, 43);
</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="gridSize">
    <size>
     <width>540</width>
     <height>66</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="deleteDialog" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>73</y>
     <width>540</width>
     <height>887</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2effffff&quot;;</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>368</y>
      <width>450</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(37, 38, 43);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteWifi">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>456</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelDele">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>456</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>383</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(37, 38, 43);</string>
    </property>
    <property name="text">
     <string>删除该热点？</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="line_2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>440</y>
      <width>450</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot; #232429&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>listWidget</zorder>
  <zorder>title</zorder>
  <zorder>back</zorder>
  <zorder>line</zorder>
  <zorder>addWifi</zorder>
  <zorder>deleteDialog</zorder>
  <zorder>dialog</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyListWidget</class>
   <extends>QListWidget</extends>
   <header>mylistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
